lib/rubycritic/source_control_systems/git.rb

```diff
@@ -21,8 +21,27 @@ def git(arg)
         self.class.git(arg)
       end
 
+      # :reek:DuplicateMethodCall
+      # :reek:NilCheck
       def self.supported?
-        git('branch 2>&1') && $CHILD_STATUS.success?
```

Comment:
JFI: This was the main reason why we could not run the RubyCritic command outside of the project; the Git class checks for the existence of a branch in the current path (pwd), and if it exists, it returns a `supported?` value as `true`.